server/src/test/java/io/crate/integrationtests/AlterTableIntegrationTest.java

```diff
@@ -400,7 +401,7 @@ public void test_increase_num_shards_does_not_delete_source_index_on_alloc_failu
         execute("alter table tbl partition (p = 1) set (\"blocks.write\" = true)");
         int numShards = (cluster().numNodes() + 1) * 2;
         assertThatThrownBy(() ->
-            execute("alter table tbl partition (p = 1) set (number_of_shards = ?)", new Object[] { numShards }, TimeValue.timeValueSeconds(30))
+            execute("alter table tbl partition (p = 1) set (number_of_shards = ?) with (timeout = '1s')", new Object[] { numShards }, TimeValue.timeValueSeconds(30))
```

Comment:
Sorry, I don't get how this tests the `with` timeout. does the 30second request timeout override the 1s?
Can we maybe have a unit test that asserts this?